src/ArrOperation.java

```diff
@@ -0,0 +1,152 @@
+public class ArrOperation
+{
+    private int array[];
+    // Parameterized constructor
+    public ArrOperation(int array[])
+    {
+        if(array.length>0)
+        {
+            this.array = array;
+        }
+        else
+        {
+            System.out.println("Can not perform operations on null array");
+            System.exit(-1);
+        }
+    }
+
+    // Getter
+    public int[] getArray()
+    {
+        return array;
+    }
+
+    // Setter
+    public void setArray(int[] array)
+    {
+        this.array = array;
+    }
```

Comment:
Can't see any uses of these method, so you could remove it.

Reply:
Removed Setter